src/game/Handlers/ChatHandler.cpp

```diff
@@ -175,8 +175,9 @@ void WorldSession::HandleMessagechatOpcode(WorldPacket& recv_data)
             lang = LANG_UNIVERSAL;
         else
         {
-            // send in universal language in two side iteration allowed mode
-            if (sWorld.getConfig(CONFIG_BOOL_ALLOW_TWO_SIDE_INTERACTION_CHAT))
+            // Send message in universal language if crossfaction chat is enabled and player is using default faction
+            // languages.
```

Comment:
why is the last word on a separate line

Reply:
Because the line had more than 120 characters.
![image](https://user-images.githubusercontent.com/8654362/111781631-9334c080-88b8-11eb-93f2-8eff91199cbf.png)

Reply:
so? whats special about that number

Reply:
Coding conventions. Originally 80 characters was the default limit, but today 120 is what most people use. It even comes marked by default (the vertical line you see in the screenshot) in CLion.